egs2/TEMPLATE/asr1/db.sh

```diff
@@ -27,6 +27,7 @@ CSJVER=dvd  ## Set your CSJ format (dvd or usb).
 CSMSC=downloads
 HKUST1=
 HKUST2=
+HUI_ACG=downloads
```

Comment:
Could you also reserve a blank entry in the JHU environment?

Reply:
Oh sorry. I forgot to add it.